docker-compose-sunbird/docker-compose.yml

```diff
@@ -163,7 +163,7 @@ services:
     volumes:
       - ./imports/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
     ports:
-      - '80:80'
```

Comment:
can we keep this 80 inside the container and 8000 to be exposed outside

Reply:
makes sense, I'll make this change

Reply:
ok